### PR TITLE
Add option to toggle visibility of cash popups

### LINKED ISSUE #1273: Feature: Option to toggle visibility of "Cash Pop-ups"
**Feature**
Whenever you make a transaction or a vehicle makes a profit, the amount of money in question appears as floating text above the location of the transaction. My suggestion is to add an option in the options menu to toggie the visibility of this, as it can become annoying when it covers large portions of the map, esepcially when you are building a lot of things in paused mode.

**Describe alternatives you've considered**
It could also be an option in the view tab (the menu that allows you to toggle height marks, invisible objects/roads, etc)

### PR DESCRIPTION
**Pull request by JimmyAllnighter**
Added a checkbox to the display options window to toggle the visibility of cash popups from construction and vehicles. Should resolve #1273.

image

### CODE REVIEW

File: src/OpenLoco/Localisation/StringIds.h [hunk 1]
```diff
@@ -1813,4 +1813,6 @@ namespace OpenLoco::StringIds
     constexpr string_id tooltip_paint_landscape_tool = 2250;
     constexpr string_id invert_right_mouse_dragging = 2251;
     constexpr string_id tooltip_invert_right_mouse_dragging = 2252;
+    constexpr string_id cash_popup_rendering = 2253;
+    constexpr string_id cash_popup_rendering_tip = 2254;
```
Comment (LeftofZen) [inline src/OpenLoco/Localisation/StringIds.h:1817]: the above few tooltips start with `tooltip_*`, so perhaps we should keep this new tooltip consistent with that

Comment (JimmyAllnighter) [reply src/OpenLoco/Localisation/StringIds.h:1817]: Good point - I'll change it to `tooltip_cash_popup_randering`

**Comment by LeftofZen** [comment]
I've tested this and everything seems to work nicely

**Review by LeftofZen — APPROVED** [review]

**Review by LeftofZen — APPROVED** [review]

**Comment by Derpercoaster** [comment]
Question, is this mean't to be in the most recent release? I downloaded it and do not see it.

**Comment by LeftofZen** [comment]
No it isn't in the latest release, it'll be in the next release

**Pull request merged**